### ISSUE TITLE
Automatic App Deployment to kubernetes cluster

### ISSUE THREAD

**Issue by jyothish-git** [argoproj/argo-cd#1730]
**Looking for an option for automatic deployment**

Hi Team,

I'm currently testing Argocd for our application deployment in Kubernetes cluster. I would like to know whether the following features are available in Argocd 
 
1. Is there any option in Argocd for creating one group and point it to a cluster.?
2. Feature like automatic deployment is possible in argocd ?

Ideally our intention is once the kubernetes cluster is up, apps will automatically be deployed without any further intervention as long as the app objects have been created in Argo and pointed at a cluster 

Any help is appreciated.
Thanks,
Jyothish

**Comment by alexec** [comment]
Try this: https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#application-of-applications

**Comment by jyothish-git** [comment]
Thanks you @alexec . I tested that with helm charts. I have one question about the projects
I created one project say "myproject" and mentioned the k8s cluster url,namespace and the git repository associated with it. So ideally when we create an application we just need to select the project name and specify the chart/yaml location from git. But every time while creating the app, its mandatory to mention server url, namespace, git location. Then what is the use of creating that project.
